web3.js/src/vote-account.ts

```diff
@@ -17,21 +17,42 @@ export type Lockout = {
 /**
  * History of how many credits earned by the end of each epoch
  */
-export type EpochCredits = {
+export type EpochCredit = {
   epoch: number;
   credits: number;
   prevCredits: number;
 };
 
+export type AuthorizedVoter = {
+  epoch: number;
+  authorizedVoter: PublicKey;
+};
+
+export type PriorVoters = {
```

Comment:
I don't think we need to expose this type, could you instead transform the circ buf into an array which starts at `idx`?

Reply:
Im not sure I fully understand what you mean by "an array which starts at idx"

Could you explain that please, thanks!

Reply:
Yeah I just mean that you could take the circ buffer into an array. The circ buffer starts at the element with index `idx` so that would be the first element of the array.

Reply:
Awesome! that makes sense.

Thanks for taking the time to look at this. Should make the amends soon!